omero/developers/build-system.txt

```diff
@@ -5,157 +5,79 @@ Build System
 
 	The page goes into details about how the build system is configured.
 
+.. _Ant: http://ant.apache.org
+.. _Ivy: http://ant.apache.org/ivy
 
-Creating binary distribution
-----------------------------
-
-The default ant target ("build-default") will build the OMERO system and
-copy the necessary components for a binary distribution to the /dist
-directory. Below is a comparison of what is taken from the build, where
-it is put, and what role it plays in the distribution.
-
-.. note::
-    By default, |OmeroCpp| is not built. Use ``build-all`` for that.
-
-.. list-table::
-    :header-rows: 1
-
-    * - OMERO_SOURCE_PREFIX
-      - OMERO_SOURCE_PREFIX/dist
-      - Comments
-    * - components/blitz/target/blitz.jar
-      - lib/server
-      - Primary Ice servants
-    * - components/blitz/target/server.jar
-      - lib/server
-      - Primary server logic
-    * - components/tools/OmeroCpp/lib*
-      - lib/
-      - Native shared libraries
-    * - components/tools/OmeroPy/build/lib
-      - lib/python
-      - Python libraries
-    * - lib/repository/<some>
-      - lib/client & lib/server
-      - Libraries needed for the build
-    * - etc/
-      - etc/
-      - Configuration
-    * - sql/\*.sql
-      - sql/
-      - SQL scripts to prepare the database
-    * - <javadoc/>
-      - docs/api
-      - (Optional) Javadocs produced with "java omero javadoc"
-
-These files are then zipped to OMERO.server-<version>.zip via "java
-omero release-zip"
-
-
-Code-signing
-^^^^^^^^^^^^
-
-The OMERO source tree includes a self-signed certificate for testing
-OMERO.insight webstart. This is not used by default to avoid problems with
-double signing official releases. The "release-webstart-signed" target can be
-used, alternatively see :source:`docs/hudson/OMERO.sh` and/or
-:ref:`server_webstart_insight_codesign`.
-
-
-Jenkins
-^^^^^^^
-
-The OME project currently uses Jenkins_ as
-a continuous integration server available :jenkins:`here <>`, so many
-binary packages can be downloaded without compiling them yourself. See the :devs_doc:`Continuous Integration documentation <ci-omero.html>` for further details.
-
-The Javadocs are always made available :javadoc:`here <>`.
-
-Build tools
------------
-
-OMERO mostly uses an `ant <http://ant.apache.org>`_-based build with
-dependency management provided by `Ivy <http://ant.apache.org/ivy>`_.
-:doc:`Native code </developers/Cpp>` is built using
-`SCons <http://scons.org>`__ and Python uses the traditional
-distutils/setuptools tools.
+OMERO mostly uses an Ant_-based build with dependency management provided by
+Ivy_. :doc:`Native code </developers/Cpp>` is built using Cmake and Python
```

Comment:
Given that the only (?) native code is C++ would the link here better be 'Cpp code', say?